hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -428,6 +428,12 @@ private void saveWorkloadProfileMetadataToInflight(WorkloadProfile profile, Hood
       HoodieCommitMetadata metadata = new HoodieCommitMetadata();
       profile.getPartitionPaths().forEach(path -> {
         WorkloadStat partitionStat = profile.getWorkloadStat(path.toString());
+        HoodieWriteStat insertStat = new HoodieWriteStat();
+        insertStat.setNumInserts(partitionStat.getNumInserts());
+        insertStat.setFileId("");
```

Comment:
Can we just leave them unset ? It should just be null..

Reply:
unset will make test fail , that cause NPE

Reply:
okay, for setPrevCommit please use HoodieWriteStat.NULL_COMMIT